### DISCUSSION TITLE
Welcome to server_bee-backend Discussions!

### DISCUSSION THREAD

**Discussion by ZingerLittleBee** [ZingerLittleBee/ServerBee#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by liling2023** [comment]
Please reconfirm input variable:
========================================================
SERVERHUB_URL: https://192.168.1.111
RECORDER_DOMAIN: 192.168.1.111
MONGO_INITDB_ROOT_USERNAME: wBTNmjnHK+t2ZNKS
MONGO_INITDB_ROOT_PASSWORD: L26UYa0QpFr4XKZr
MONGODB_URI: mongodb://wBTNmjnHK+t2ZNKS:L26UYa0QpFr4XKZr@mongo:27017/
NEXTAUTH_SECRET: Sl7J+CLDMqUnuFQI3fXd0NLOcMUdzEzE1yoaGxNUHJw=
SERVER_JWT_SECRET: o1reMyAbjK26jZBa6KGeY90DGvu80JprNSfsJRO8GHU=
========================================================
Installation will start after confirmation.
Are the variables correct? (y/n):
y
正克隆到 'server_bee-backend'...
remote: Enumerating objects: 4968, done.
remote: Counting objects: 100% (991/991), done.
remote: Compressing objects: 100% (528/528), done.
remote: Total 4968 (delta 495), reused 778 (delta 436), pack-reused 3977
接收对象中: 100% (4968/4968), 9.29 MiB | 2.86 MiB/s, 完成.
处理 delta 中: 100% (2840/2840), 完成.
WARNING: The SERVER_TOKEN variable is not set. Defaulting to a blank string.
Creating network "docker_default" with the default driver
Pulling caddy (caddy:2-alpine)...
ERROR: Get "https://registry-1.docker.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
Installation completed.
 If you are login for the first time, please route to  localhost/init to create an admin account.

**Reply by ZingerLittleBee** [reply]
check docker network, try again

**Reply by liling2023** [reply]
通过多次切换代理，安装成功了，同时又出现了新的问题。
我是安装在本地Centos9服务器上 的，在一键安装的时候，配置的domain是ip地址。
第一个问题https://192.168.1.111,第二个问题192.168.1.111/hub,然后Enter，第四个就是n，然后最终输入y。
安装完成后，设定admin账号密码后，登录提示
![微信截图_20240405104508](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/d53ad31e-abb3-4c58-a7c9-b80bcf88232e)
![微信截图_20240405104640](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/0748bcc6-19b1-4ad9-9607-28450e447d34)

**Reply by liling2023** [reply]
openssl有安装

**Reply by ZingerLittleBee** [reply]
下列三种方法任选其一

- 使用 http 访问
- 使用 cloudflare 打开小云朵来支持 https 访问
- 使用 caddy 来自动申请 ssl 证书

**Reply by liling2023** [reply]
感谢

**Reply by liling2023** [reply]
作为新手的我，又遇到新的问题了。
服务是搭建起来了，添加了客户端，但是获取不到信息，包括服务器本机的信息也没获取到
我查看了，serverbee_web这个没有运行起来，状态只是stared，不是runing。
同时我也看了docker-compose.yml文件中，web是存在的.
期间重新安装过多次，切换过多次域名，ip地址，问题依旧。

**Reply by liling2023** [reply]
![微信图片_20240406083131](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/49de0237-446e-45f1-ad43-dd6bc1e80ca8)
[](url)
![微信截图_20240406083155](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/2ad373e5-0915-4ffc-b762-dd78633cac0c)

**Reply by liling2023** [reply]
Select service to check logs:
1) serverhub
2) serverbee-recorder
3) serverbee-interactor
4) serverbee-web
5) serverbee-caddy
6) serverbee-mongo
7) back
#? 4
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]
error: unexpected argument '-h' found

Usage: serverbee-web [OPTIONS]


删除-h参数，也没效果

**Reply by liling2023** [reply]
删除-h提示
error: unexpected argument 'http://recorder:9528' found

Usage: serverbee-web [OPTIONS]

**Comment by liling2023** [comment]
![微信截图_20240405104740](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/40e40925-9ee8-49d3-95a1-7a2616888984)
![微信截图_20240405104803](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/c880f684-b281-4582-9852-aea69966ad47)

**Reply by ZingerLittleBee** [reply]
serverbee-web 服务自带一个 web 界面，访问 localhost:9527 ，在设置中就可以设置参数，
具体参考[文档](https://docs.serverhub.app/zh-CN/getting-started/configuration#%E6%B7%BB%E5%8A%A0%E6%9C%8D%E5%8A%A1%E5%99%A8)

**Reply by liling2023** [reply]
我有单独尝试安装web，但是hub数据还是接收不到

**Reply by ZingerLittleBee** [reply]
docker 镜像需要重新拉去下，更新了环境变量

**Reply by liling2023** [reply]
目前已知问题：
1.windows版本，最新客户端，无法加入hub。(似乎端口格式不被支持)
2.recorder服务，不能配置为ip地址。
![微信截图_20240406170738](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/8a91ee9e-b5f8-4e64-99b0-57e8e06f0cd5)

**Reply by ZingerLittleBee** [reply]
你截图的 serverbee-web 不是最新版的，删除 docker image 重新拉取

**Reply by liling2023** [reply]
截图是windows版本的2.21版本。
docke下安装的web是可以正常添加。
请问下，这个host是不是填写服务器的地址？
hub本机，默认填写的是recorder:9528，目前我只有本机能获取到信息

**Reply by liling2023** [reply]
我重新测试安装了一套centos下的docker 下的 zingerbee/serverbee-web。可以正常获取到信息。
目前看来只有windows版本milk版本适配有点小问题了
![微信截图_20240406233351](https://github.com/ZingerLittleBee/server_bee-backend/assets/124797335/8964343b-ce5f-453a-bc29-8255d0660a23)

**Reply by ZingerLittleBee** [reply]
servermlik 还没更新，因为还不是正式版
> hub本机，默认填写的是recorder:9528，目前我只有本机能获取到信息
`recorder:9528` 只能在 docker-compose 部署的才可以访问
host 字段 -> server_url (.env.exmaple 中有示例)

**Reply by liling2023** [reply]
嘿嘿。请问有没有更新计划呀。最近准备上这套监控平台。
蛮多都是windows服务器的

**Reply by ZingerLittleBee** [reply]
一直在更新，看 commit
不过到发布正式版还需要一段时间
